app/src/main/java/com/mapbox/maps/testapp/examples/AnimatedMarkerActivity.kt

```diff
@@ -75,15 +75,11 @@ class AnimatedMarkerActivity : AppCompatActivity(), OnMapClickListener {
         startValue.latitude() + fraction * (endValue.latitude() - startValue.latitude())
       )
     }
-    var tick = 0
     animator = ValueAnimator().apply {
       setObjectValues(currentPoint, point)
       setEvaluator(pointEvaluator)
       addUpdateListener {
-        // TODO remove throttle when https://github.com/mapbox/mapbox-maps-android/issues/554 fixed
-        if (++tick % 2 == 0) {
-          geojsonSource.geometry(it.animatedValue as Point)
-        }
```

Comment:
by processing the GeojsonSouce async on platform, does it solve the issue that frequent GeoJsonSource update not showing up in the UI?

Reply:
Basically it's already fixed in latest `main`. But this async mode should work even better I guess as we use single worker thread and because of that we preserve correct order of updates.

Reply:
👍 this is a great fix